control-plane/connect-inject/webhook/mesh_webhook.go

```diff
@@ -306,6 +311,7 @@ func (w *MeshWebhook) Handle(ctx context.Context, req admission.Request) admissi
 			w.Log.Error(err, "error configuring injection sidecar container", "request name", req.Name)
 			return admission.Errored(http.StatusInternalServerError, fmt.Errorf("error configuring injection sidecar container: %s", err))
 		}
+		// TODO: invert to start the Envoy sidecar before the application container
```

Comment:
This is out of scope for this PR, just leaving a note for future work on proxy lifecycle graceful startup.